google/cloud/forseti/common/gcp_api/storage.py

```diff
@@ -442,13 +471,21 @@ def get_default_object_acls(self, bucket):
         """
         try:
             results = self.repository.default_object_acls.list(resource=bucket)
-            return api_helpers.flatten_list_results(results, 'items')
+            flattened_results = api_helpers.flatten_list_results(results,
+                                                                 'items')
+            LOGGER.debug("Getting default object acls for GCS bucket, bucket"
+                         " = %s, flattened_results = %s",
+                         bucket, flattened_results)
+            return flattened_results
         except (errors.HttpError, HttpLib2Error) as e:
-            LOGGER.warn(api_errors.ApiExecutionError(bucket, e))
-            raise api_errors.ApiExecutionError('defaultObjectAccessControls', e)
+            raise api_errors.ApiExecutionError(
+                API_EXECUTION_ERROR_ARG_FORMAT.format(
+                    'defaultObjectAccessControls',
+                    'bucket',
+                    bucket), e)
 
     def get_objects(self, bucket):
-        """Gets all GCS buckets for a project.
+        """Gets all objects in a bucket
```

Comment:
First line comments should always end in .

Reply:
fixed.